src/common/variables.c

```diff
@@ -423,8 +494,8 @@ static char *get_base_value(dt_variables_params_t *params, char **variable)
         break;
     }
   }
-  else if((has_prefix(variable, "LABELS_ICONS") ||
-           has_prefix(variable, "LABELS_COLORICONS"))
+  else if((_has_prefix(variable, "LABELS_ICONS") ||
```

Comment:
likewise : `LABELS.ICONS` & `LABELS.COLORICONS`.

Reply:
Do you mean that all '_' should become '.' ?

Reply:
Yes!

Reply:
Done. I haven't included DATE trying to keep things as short as possible.

Reply:
I've also grouped MAKER, MODEL and LENS under EXIF, but that may be questionable.